<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Structured15DGridWithCrossSectionCrossSectionWindow</class>
 <widget class="QMainWindow" name="Structured15DGridWithCrossSectionCrossSectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Structured15DGridWithCrossSectionCrossSectionWindowGraphicsView" name="graphicsView" native="true"/>
      <widget class="Structured15DGridWithCrossSectionCrossSectionWindowTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Structured15DGridWithCrossSectionCrossSectionWindowGraphicsView</class>
   <extends>QWidget</extends>
   <header>subwindow/structured15dgridwithcrosssectioncrosssectionwindow/structured15dgridwithcrosssectioncrosssectionwindowgraphicsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Structured15DGridWithCrossSectionCrossSectionWindowTableView</class>
   <extends>QTableWidget</extends>
   <header>subwindow/structured15dgridwithcrosssectioncrosssectionwindow/structured15dgridwithcrosssectioncrosssectionwindowtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
